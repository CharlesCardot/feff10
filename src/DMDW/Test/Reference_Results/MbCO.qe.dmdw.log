Warning: The dynamical matrix is not symmetric
         All results should be carefully examined
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -75.5424661284
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -73.2202171403
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -69.9599189363
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -73.2202196803
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -66.8398821224
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -66.8398821224
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -61.9306155977
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -56.4851927274
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -63.9157477219
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -63.9157477219
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -56.4851927274
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -61.9306155977
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -97.0792809987
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -97.0792809987
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -92.5487519720
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -92.5487519720
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -81.3698405325
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -81.3698405325
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -80.7939868715
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -79.4667951194
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -80.4021183460
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -80.4021183460
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -79.4667951194
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -80.7939868715
 Warning: The number of poles found is not equal
          to the number of iterations
nPoles =     3
Warning: There is an imaginary frequency.
         Setting weight to 0:     0     -131.8300092543
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -82.7430103452
 Warning: The number of poles found is not equal
          to the number of iterations
nPoles =     3
Warning: There is an imaginary frequency.
         Setting weight to 0:     0     -131.8300092543
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -82.7430103452
Warning: There is an imaginary frequency.
         Setting weight to 0:     0     -127.1066295963
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -80.0265727613
Warning: There is an imaginary frequency.
         Setting weight to 0:     0     -127.1066295963
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -80.0265727613
Warning: There is an imaginary frequency.
         Setting weight to 0:     0     -110.8449490326
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -70.4766176383
Warning: There is an imaginary frequency.
         Setting weight to 0:     0     -110.8449490326
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -70.4766176383
Warning: There is an imaginary frequency.
         Setting weight to 0:     0     -114.7008062601
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -73.0862724388
Warning: There is an imaginary frequency.
         Setting weight to 0:     0     -111.7264469082
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -70.8763139794
Warning: There is an imaginary frequency.
         Setting weight to 0:     0     -113.0545901688
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -72.3748217383
Warning: There is an imaginary frequency.
         Setting weight to 0:     0     -113.0545901688
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -72.3748217383
Warning: There is an imaginary frequency.
         Setting weight to 0:     0     -111.7264469082
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -70.8763139794
Warning: There is an imaginary frequency.
         Setting weight to 0:     0     -114.7008062601
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -73.0862724388
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -72.6708157475
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -16.8034849942
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -55.0493967450
 Warning: The number of poles found is not equal
          to the number of iterations
nPoles =     3
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -68.1137263019
 Warning: The number of poles found is not equal
          to the number of iterations
nPoles =     3
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -75.7107787564
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -34.1634706383
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -34.3394375629
Warning: There is an imaginary frequency.
         Setting weight to 0:     0     -122.3300557571
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -69.5815829469
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -78.0529416755
 Warning: The number of poles found is not equal
          to the number of iterations
nPoles =     3
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -84.1116179122
 Warning: The number of poles found is not equal
          to the number of iterations
nPoles =     3
Warning: There is an imaginary frequency.
         Setting weight to 0:     0      -94.1045012584
Warning: There is an imaginary frequency.
         Setting weight to 0:     1      -21.1276818175
