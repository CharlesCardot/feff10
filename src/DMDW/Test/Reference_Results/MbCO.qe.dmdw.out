# Lanczos recursion order:    4
# Temperature:   77.00
# Dynamical matrix file: MbCO.qe.dym

--------------------------------------------------------------
 Path Indices:    1   2
 pDOS Centroid (THz):   21.355
 pDOS Poles:     Freq. (THz)    Weight
                 -75.542       0.00000
                  16.222       0.90625
                  52.907       0.08775
                 115.487       0.00033
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.016   2.596
--------------------------------------------------------------
 Path Indices:    1   3
 pDOS Centroid (THz):   23.623
 pDOS Poles:     Freq. (THz)    Weight
                 -73.220       0.00000
                  17.350       0.87822
                  54.497       0.11259
                 113.772       0.00047
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.015   2.378
--------------------------------------------------------------
 Path Indices:    1   4
 pDOS Centroid (THz):   24.727
 pDOS Poles:     Freq. (THz)    Weight
                 -69.960       0.00000
                   5.086       0.70627
                  47.600       0.28206
                 110.034       0.00066
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.018   7.084
--------------------------------------------------------------
 Path Indices:    1   5
 pDOS Centroid (THz):   23.623
 pDOS Poles:     Freq. (THz)    Weight
                 -73.220       0.00000
                  17.350       0.87822
                  54.497       0.11259
                 113.772       0.00047
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.015   2.378
--------------------------------------------------------------
 Path Indices:    1   6
 pDOS Centroid (THz):   28.358
 pDOS Poles:     Freq. (THz)    Weight
                 -66.840       0.00000
                   6.917       0.64726
                  49.847       0.33662
                  99.857       0.00061
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.053   5.260
--------------------------------------------------------------
 Path Indices:    1   7
 pDOS Centroid (THz):   28.358
 pDOS Poles:     Freq. (THz)    Weight
                 -66.840       0.00000
                   6.917       0.64726
                  49.847       0.33662
                  99.857       0.00061
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.053   5.260
--------------------------------------------------------------
 Path Indices:    1   8
 pDOS Centroid (THz):   29.134
 pDOS Poles:     Freq. (THz)    Weight
                 -61.931       0.00000
                  15.491       0.69287
                  51.019       0.28842
                 107.532       0.00023
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.051   2.576
--------------------------------------------------------------
 Path Indices:    1   9
 pDOS Centroid (THz):   29.330
 pDOS Poles:     Freq. (THz)    Weight
                 -56.485       0.00000
                  21.982       0.77719
                  52.918       0.19928
                 114.890       0.00011
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.051   2.000
--------------------------------------------------------------
 Path Indices:    1  10
 pDOS Centroid (THz):   29.189
 pDOS Poles:     Freq. (THz)    Weight
                 -63.916       0.00000
                  22.505       0.80443
                  52.602       0.18124
                 118.923       0.00009
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.054   2.004
--------------------------------------------------------------
 Path Indices:    1  11
 pDOS Centroid (THz):   29.189
 pDOS Poles:     Freq. (THz)    Weight
                 -63.916       0.00000
                  22.505       0.80443
                  52.602       0.18124
                 118.923       0.00009
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.054   2.004
--------------------------------------------------------------
 Path Indices:    1  12
 pDOS Centroid (THz):   29.330
 pDOS Poles:     Freq. (THz)    Weight
                 -56.485       0.00000
                  21.982       0.77719
                  52.918       0.19928
                 114.890       0.00011
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.051   2.000
--------------------------------------------------------------
 Path Indices:    1  13
 pDOS Centroid (THz):   29.134
 pDOS Poles:     Freq. (THz)    Weight
                 -61.931       0.00000
                  15.491       0.69287
                  51.019       0.28842
                 107.532       0.00023
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.051   2.576
--------------------------------------------------------------
 Path Indices:    1  14
 pDOS Centroid (THz):   28.437
 pDOS Poles:     Freq. (THz)    Weight
                 -97.079       0.00000
                  18.153       0.87182
                  67.230       0.10450
                 115.745       0.01193
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.429   2.540
--------------------------------------------------------------
 Path Indices:    1  15
 pDOS Centroid (THz):   28.437
 pDOS Poles:     Freq. (THz)    Weight
                 -97.079       0.00000
                  18.153       0.87182
                  67.230       0.10450
                 115.745       0.01193
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.429   2.540
--------------------------------------------------------------
 Path Indices:    1  16
 pDOS Centroid (THz):   28.944
 pDOS Poles:     Freq. (THz)    Weight
                 -92.549       0.00000
                  20.204       0.88911
                  69.985       0.08344
                 112.931       0.01413
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.429   2.317
--------------------------------------------------------------
 Path Indices:    1  17
 pDOS Centroid (THz):   28.944
 pDOS Poles:     Freq. (THz)    Weight
                 -92.549       0.00000
                  20.204       0.88911
                  69.985       0.08344
                 112.931       0.01413
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.429   2.317
--------------------------------------------------------------
 Path Indices:    1  18
 pDOS Centroid (THz):   29.656
 pDOS Poles:     Freq. (THz)    Weight
                 -81.370       0.00000
                  23.591       0.87407
                  67.245       0.08776
                 114.659       0.01259
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.281   1.967
--------------------------------------------------------------
 Path Indices:    1  19
 pDOS Centroid (THz):   29.656
 pDOS Poles:     Freq. (THz)    Weight
                 -81.370       0.00000
                  23.591       0.87407
                  67.245       0.08776
                 114.659       0.01259
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.281   1.967
--------------------------------------------------------------
 Path Indices:    1  20
 pDOS Centroid (THz):   29.826
 pDOS Poles:     Freq. (THz)    Weight
                 -80.794       0.00000
                  22.553       0.85436
                  65.327       0.10796
                 115.118       0.01214
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.280   2.027
--------------------------------------------------------------
 Path Indices:    1  21
 pDOS Centroid (THz):   29.843
 pDOS Poles:     Freq. (THz)    Weight
                 -79.467       0.00000
                  22.685       0.84989
                  65.124       0.10997
                 115.123       0.01228
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.280   2.007
--------------------------------------------------------------
 Path Indices:    1  22
 pDOS Centroid (THz):   29.680
 pDOS Poles:     Freq. (THz)    Weight
                 -80.402       0.00000
                  23.451       0.86546
                  67.327       0.09306
                 114.615       0.01282
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.282   1.963
--------------------------------------------------------------
 Path Indices:    1  23
 pDOS Centroid (THz):   29.680
 pDOS Poles:     Freq. (THz)    Weight
                 -80.402       0.00000
                  23.451       0.86546
                  67.327       0.09306
                 114.615       0.01282
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.282   1.963
--------------------------------------------------------------
 Path Indices:    1  24
 pDOS Centroid (THz):   29.843
 pDOS Poles:     Freq. (THz)    Weight
                 -79.467       0.00000
                  22.685       0.84989
                  65.124       0.10997
                 115.123       0.01228
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.280   2.007
--------------------------------------------------------------
 Path Indices:    1  25
 pDOS Centroid (THz):   29.826
 pDOS Poles:     Freq. (THz)    Weight
                 -80.794       0.00000
                  22.553       0.85436
                  65.327       0.10796
                 115.118       0.01214
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.280   2.027
--------------------------------------------------------------
 Path Indices:    1  26
 pDOS Centroid (THz):   76.381
 pDOS Poles:     Freq. (THz)    Weight
                -131.830       0.00000
                 -82.743       0.00000
                 103.573       0.67810
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.520   3.342
--------------------------------------------------------------
 Path Indices:    1  27
 pDOS Centroid (THz):   76.381
 pDOS Poles:     Freq. (THz)    Weight
                -131.830       0.00000
                 -82.743       0.00000
                 103.573       0.67810
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.520   3.342
--------------------------------------------------------------
 Path Indices:    1  28
 pDOS Centroid (THz):   78.152
 pDOS Poles:     Freq. (THz)    Weight
                -127.107       0.00000
                 -80.027       0.00000
                 102.497       0.69449
                 139.935       0.03734
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.520   3.595
--------------------------------------------------------------
 Path Indices:    1  29
 pDOS Centroid (THz):   78.152
 pDOS Poles:     Freq. (THz)    Weight
                -127.107       0.00000
                 -80.027       0.00000
                 102.497       0.69449
                 139.935       0.03734
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.520   3.595
--------------------------------------------------------------
 Path Indices:    1  30
 pDOS Centroid (THz):   79.019
 pDOS Poles:     Freq. (THz)    Weight
                -110.845       0.00000
                 -70.477       0.00000
                  95.821       0.60965
                 120.289       0.14408
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.255   3.859
--------------------------------------------------------------
 Path Indices:    1  31
 pDOS Centroid (THz):   79.019
 pDOS Poles:     Freq. (THz)    Weight
                -110.845       0.00000
                 -70.477       0.00000
                  95.821       0.60965
                 120.289       0.14408
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.255   3.859
--------------------------------------------------------------
 Path Indices:    1  32
 pDOS Centroid (THz):   77.933
 pDOS Poles:     Freq. (THz)    Weight
                -114.701       0.00000
                 -73.086       0.00000
                  96.726       0.62773
                 122.858       0.11708
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.254   3.799
--------------------------------------------------------------
 Path Indices:    1  33
 pDOS Centroid (THz):   78.827
 pDOS Poles:     Freq. (THz)    Weight
                -111.726       0.00000
                 -70.876       0.00000
                  96.006       0.61221
                 121.011       0.13884
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.254   3.841
--------------------------------------------------------------
 Path Indices:    1  34
 pDOS Centroid (THz):   78.449
 pDOS Poles:     Freq. (THz)    Weight
                -113.055       0.00000
                 -72.375       0.00000
                  96.309       0.61916
                 121.759       0.13091
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.257   3.830
--------------------------------------------------------------
 Path Indices:    1  35
 pDOS Centroid (THz):   78.449
 pDOS Poles:     Freq. (THz)    Weight
                -113.055       0.00000
                 -72.375       0.00000
                  96.309       0.61916
                 121.759       0.13091
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.257   3.830
--------------------------------------------------------------
 Path Indices:    1  36
 pDOS Centroid (THz):   78.827
 pDOS Poles:     Freq. (THz)    Weight
                -111.726       0.00000
                 -70.876       0.00000
                  96.006       0.61221
                 121.011       0.13884
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.254   3.841
--------------------------------------------------------------
 Path Indices:    1  37
 pDOS Centroid (THz):   77.933
 pDOS Poles:     Freq. (THz)    Weight
                -114.701       0.00000
                 -73.086       0.00000
                  96.726       0.62773
                 122.858       0.11708
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.254   3.799
--------------------------------------------------------------
 Path Indices:    1  38
 pDOS Centroid (THz):   31.321
 pDOS Poles:     Freq. (THz)    Weight
                 -72.671       0.00000
                  28.811       0.89144
                  53.948       0.09809
                 106.774       0.00065
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.101   1.479
--------------------------------------------------------------
 Path Indices:    1  39
 pDOS Centroid (THz):   34.501
 pDOS Poles:     Freq. (THz)    Weight
                 -16.803       0.00000
                  39.796       0.56241
                  66.470       0.08936
                 130.180       0.00018
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.073   0.791
--------------------------------------------------------------
 Path Indices:    1  40
 pDOS Centroid (THz):   29.924
 pDOS Poles:     Freq. (THz)    Weight
                 -55.049       0.00000
                  21.484       0.76066
                  66.470       0.16924
                 128.883       0.00046
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.145   1.941
--------------------------------------------------------------
 Path Indices:    1  41
 pDOS Centroid (THz):   34.873
 pDOS Poles:     Freq. (THz)    Weight
                 -68.114       0.00000
                  33.083       0.95084
                 114.833       0.00840
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.247   1.300
--------------------------------------------------------------
 Path Indices:    1  42
 pDOS Centroid (THz):   38.684
 pDOS Poles:     Freq. (THz)    Weight
                 -75.711       0.00000
                  36.489       0.94731
                 106.277       0.01976
 Path Length (Ang), s^2 (1e-3 Ang^2):  4.304   1.337
--------------------------------------------------------------
 Path Indices:    1  43
 pDOS Centroid (THz):   34.785
 pDOS Poles:     Freq. (THz)    Weight
                 -34.163       0.00000
                  30.381       0.51750
                  63.618       0.24786
                 127.930       0.00017
 Path Length (Ang), s^2 (1e-3 Ang^2):  1.731   1.070
--------------------------------------------------------------
 Path Indices:    1  44
 pDOS Centroid (THz):   37.104
 pDOS Poles:     Freq. (THz)    Weight
                 -34.339       0.00000
                  23.408       0.58004
                  54.970       0.36978
                 127.399       0.00004
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.897   1.280
--------------------------------------------------------------
 Path Indices:    1  45
 pDOS Centroid (THz):   53.432
 pDOS Poles:     Freq. (THz)    Weight
                -122.330       0.00000
                 -69.582       0.00000
                  81.668       0.61969
                 131.341       0.03982
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.322   4.028
--------------------------------------------------------------
 Path Indices:    1  46
 pDOS Centroid (THz):   47.611
 pDOS Poles:     Freq. (THz)    Weight
                 -78.053       0.00000
                  33.870       0.69746
                  78.019       0.17447
                 106.109       0.06829
 Path Length (Ang), s^2 (1e-3 Ang^2):  3.296  11.981
--------------------------------------------------------------
 Path Indices:    1  47
 pDOS Centroid (THz):  118.563
 pDOS Poles:     Freq. (THz)    Weight
                 -84.112       0.00000
                  14.666       0.22887
                 138.602       0.61650
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.157  10.238
--------------------------------------------------------------
 Path Indices:    1  48
 pDOS Centroid (THz):  110.174
 pDOS Poles:     Freq. (THz)    Weight
                 -94.105       0.00000
                 -21.128       0.00000
                 130.490       0.65164
 Path Length (Ang), s^2 (1e-3 Ang^2):  5.332   2.549
